package.json

```diff
@@ -31,35 +30,34 @@
         "babel-polyfill": "^6.26.0",
         "babel-runtime": "^6.20.0",
         "create-react-class": "^15.6.2",
-        "debug": "^3.0.1",
+        "debug": "^4.0.1",
         "fs-extra": "^7.0.0",
         "keymirror": "^0.1.1",
         "prop-types": "^15.6.0",
-        "react": "16.4.2",
-        "react-native": "^0.56.0",
-        "react-navigation": "2.12.1",
-        "react-redux": "5.07",
+        "react": "^16.5.1",
+        "react-devtools": "^3.3.4",
+        "react-native": "^0.57.0",
+        "react-navigation": "2.14.2",
+        "react-redux": "5.0.7",
         "redux": "4.0.0",
         "redux-thunk": "2.3.0",
-        "styled-components": "^3.3.0"
+        "styled-components": "^3.4.9"
     },
     "devDependencies": {
         "babel-eslint": "^9.0.0",
-        "babel-jest": "23.4.2",
+        "babel-jest": "23.6.0",
         "babel-plugin-globals": "^3.0.0",
         "babel-plugin-module-resolver": "^3.0.0",
-        "babel-preset-react-native": "^5.0.0",
-        "eslint": "^5.4.0",
+        "babel-preset-react-native": "^4.0.0",
```

Comment:
This one moved down?

Reply:
The next release isn't officially out and the updates allow for us to move back to the stable release.